Packages.props

```diff
@@ -1,23 +1,23 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 	<ItemGroup>
-    	<PackageReference Update="BenchmarkDotNet" Version="[0.13.1]" />
-		<PackageReference Update="D2L.CodeStyle.Analyzers" Version="[0.142.0]" />
-		<PackageReference Update="D2L.CodeStyle.Annotations" Version="[0.28.0]" />
-		<PackageReference Update="D2L.Services.Core.Exceptions" Version="[2.0.1.15]" />
-		<PackageReference Update="FluentAssertions" Version="[6.2.0]" />
+    	<PackageReference Update="BenchmarkDotNet" Version="0.13.1" />
+		<PackageReference Update="D2L.CodeStyle.Analyzers" Version="0.142.0" />
+		<PackageReference Update="D2L.CodeStyle.Annotations" Version="0.28.0" />
+		<PackageReference Update="D2L.Services.Core.Exceptions" Version="2.0.1.15" />
+		<PackageReference Update="FluentAssertions" Version="6.2.0" />
 		<PackageReference Update="Microsoft.AspNet.WebApi.Core" Version="[5.2.7]" />
 		<PackageReference Update="Microsoft.AspNet.WebApi.OwinSelfHost" Version="[5.2.7]" />
```

Comment:
Keeping the old-school WebAPI references pinned because of framework constraints.
(This package doesn't exist at all in .NET Core)